.github/workflows/kitchen.windows.yml

```diff
@@ -0,0 +1,121 @@
+---
```

Comment:
I'd prefer this go in .github/workflows/templates/windows.yml and end up part of the generated .github/workfows/main.yml similar to the lint.yml probably, but run at the same stage as the linux jobs.

Windows tests will be really nice to have.

Reply:
@bryceml Doesn't the same apply to #1548 as well?  Both PRs are using `strategy:matrix` so would have to be coerced in the way you've mentioned about `lint.yml`.

> but run at the same stage as the linux jobs

They run concurrently alongside the Linux jobs already.  I'm pretty sure having them inside the same workflow run will actually be slower overall.

Reply:
mainly I would want these not to run if lint doesn't pass.  github limits the number of concurrent things we can run at once, and I'd prefer not to waste that time if it doesn't pass lint (shellcheck).

Yeah, applies to #1548 too.